docs/installation/getting-started/quick-start.md

```diff
@@ -4,318 +4,1720 @@
-	entry: './app.js',
+```js
+CKEDITOR.ClassicEditor
+	.create( document.querySelector( '#editor' ), {
+		removePlugins: [ 'Comments', 'TrackChanges', 'TrackChangesData', 'RevisionHistory', 'RealTimeCollaborativeComments', 'RealTimeCollaborativeTrackChanges', 'RealTimeCollaborativeRevisionHistory', 'RealTimeCollaborativeEditing', 'PresenceList', 'ExportWord', 'ExportPdf', 'Pagination', 'WProofreader', 'MathType' ],
+		toolbar: {
+			items: [
+				'undo', 'redo',
+				'|',
+				'findAndReplace', 'selectAll',
+				'|',
+				'heading',
+				'|',
+				'removeFormat', 'bold', 'italic', 'strikethrough', 'underline', 'code', 'subscript', 'superscript',
+				'|',
+				'specialCharacters', 'horizontalLine', 'pageBreak',
+				'|',
+				'-',
+				'highlight', 'fontSize', 'fontFamily', 'fontColor', 'fontBackgroundColor',
+				'|',
+				'link', 'blockQuote', 'insertTable', 'uploadImage', 'mediaEmbed', 'codeBlock', 'htmlEmbed',
+				'|',
+				'bulletedList', 'numberedList', 'todoList',
+				'|',
+				'outdent', 'indent', 'alignment',
+				'|',
+				'textPartLanguage',
+				'|',
+				'sourceEditing'
+			],
+			shouldNotGroupWhenFull: true
+		},
+	image: {
+			styles: [
+				'alignCenter',
+				'alignLeft',
+				'alignRight'
+			],
+			resizeOptions: [
+				{
+					name: 'resizeImage:original',
+					label: 'Original',
+					value: null
+				},
+				{
+					name: 'resizeImage:50',
+					label: '50%',
+					value: '50'
+				},
+				{
+					name: 'resizeImage:75',
+					label: '75%',
+					value: '75'
+				}
+			],
+			toolbar: [
+				'imageTextAlternative', 'toggleImageCaption', '|',
+				'imageStyle:inline', 'imageStyle:wrapText', 'imageStyle:breakText', 'imageStyle:side', '|',
+				'resizeImage'
+			],
+			insert: {
+				integrations: [
+					'insertImageViaUrl'
+				]
+			}
+		},
+		list: {
+			properties: {
+				styles: true,
+				startIndex: true,
+				reversed: true
+			}
+		},
+		link: {
+			decorators: {
+				addTargetToExternalLinks: true,
+				defaultProtocol: 'https://',
+				toggleDownloadable: {
+					mode: 'manual',
+					label: 'Downloadable',
+					attributes: {
+						download: 'file'
+					}
+				}
+			}
+		},
+		mention: {
+			feeds: [
+				{
+					marker: '@',
+					feed: [
+						'@apple', '@bears', '@brownie', '@cake', '@cake', '@candy', '@canes', '@chocolate', '@cookie', '@cotton', '@cream',
+						'@cupcake', '@danish', '@donut', '@dragée', '@fruitcake', '@gingerbread', '@gummi', '@ice', '@jelly-o',
+						'@liquorice', '@macaroon', '@marzipan', '@oat', '@pie', '@plum', '@pudding', '@sesame', '@snaps', '@soufflé',
+						'@sugar', '@sweet', '@topping', '@wafer'
+					],
+					minimumCharacters: 1
+				}
+			]
+		},
+		placeholder: 'Type or paste your content here!',
+		table: {
+			contentToolbar: [
+				'tableColumn', 'tableRow', 'mergeTableCells', 'tableProperties', 'tableCellProperties', 'toggleTableCaption'
+			]
+		},
+	} )
+	.catch( error => {
+		console.log( error );
+	} );
+```
```

Comment:
In general, I agree that step by step attitude and in the end presenting full implementation is good, but in this case, pasting a large amount of source code to say that plugins can be removed and just after that place another large amount of almost the same code with full implementation looks like too much. 
I would suggest removing this snippet and within the part about eliminating plugins and customising the toolbar refer to the full implementation below.